SCClassLibrary/Common/UnitTesting/UnitTest.sc

```diff
@@ -208,16 +208,18 @@ UnitTest {
 	// waits for condition with a maxTime limit
 	// if time expires, the test is a failure
 	wait { |condition, failureMessage, maxTime = 10.0|
-		var limit = maxTime / 0.05;
+
+		var limit = max(maxTime / 0.05, 1);
+		var dt = min(maxTime, 0.1);
```

Comment:
So, using the argument default, that's `min(10.0, 0.1)`... that doesn't seem right. Should be `maxTime / limit`, no? (Requiring limit >= 1 is sensible, but then that should feed back into dt somehow.)

Reply:
@jamshark70 I've just pushed an improved version of UnitTest wait. 

Of course, all this has nothing to do with the pull request at hand, but well if we have to fix it to get there, we have to fix it to get there.

Reply:
ping …